src/libbson/tests/json/bson_corpus/code_w_scope.json

```diff
@@ -26,7 +26,7 @@
         {
             "description": "Unicode and embedded null in code string, empty scope",
             "canonical_bson": "1A0000000F61001200000005000000C3A9006400050000000000",
-            "canonical_extjson": "{\"a\" : {\"$code\" : \"é\\u0000d\", \"$scope\" : {}}}"
+            "canonical_extjson": "{\"a\" : {\"$code\" : \"\\u00e9\\u0000d\", \"$scope\" : {}}}"
```

Comment:
Note: this test was already skipped in `test-bson-corpus.c` due to [CDRIVER-1879](https://jira.mongodb.org/browse/CDRIVER-1879) (resolved as "Won't Fix").